### PR TITLE
Add networkID to process-agent checks

### PR DESCRIPTION
**Pull request by leeavital**
### What does this PR do?
Include networkId in process/container/connection check.

Depends on https://github.com/DataDog/agent-payload/pull/23

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: pkg/process/checks/container.go [hunk 1]
```diff
@@ -58,6 +59,11 @@ func (c *ContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Me
 		return nil, err
 	}
 
+	networkID, err := util.GetNetworkID()
```
Comment (sunhay) [inline pkg/process/checks/container.go]: Could we possibly get this in `Init()` or is there caching and race conditions at play?

Comment (leeavital) [reply pkg/process/checks/container.go]: We could get it in `Init()`.

**Review by sunhay — DISMISSED** [review]
LGTM. Just one question.

File: pkg/process/checks/container.go [hunk 2]
```diff
@@ -30,11 +31,18 @@ type ContainerCheck struct {
...
+	networkID       string
 }
 
 // Init initializes a ContainerCheck instance.
 func (c *ContainerCheck) Init(cfg *config.AgentConfig, info *model.SystemInfo) {
 	c.sysInfo = info
+
+	networkID, err := agentutil.GetNetworkID()
+	if err != nil {
+		log.Infof("no network ID detected: %s", err)
```
Comment (sunhay) [inline pkg/process/checks/container.go]: Nice: this looks useful as `INFO`

**Review by sunhay — APPROVED** [review]

**Comment by sunhay** [comment]
Should also probably add a `reno` changelog here

**Comment by leeavital** [comment]
@sunhay there's an entry in the change that added `GetNetworkID()` which I think covers this.

**Pull request merged**